### PR TITLE
vote: kennytm as TiKV committer

### PR DESCRIPTION
**Pull request by overvenus**
vote: kennytm as TiKV committer

### CODE REVIEW

**Review by tisonkun — APPROVED** [review]
+1 (non-binding). @kennytm well deserved!

**Comment by skyzh** [comment]
+1 for this

**Comment by tisonkun** [comment]
@overvenus you may also update the membership file.

@skyzh thanks for your participant. You may directly use approval button rather than leave comment. Though both count effectively :P

**Review by BusyJay — APPROVED** [review]
Thanks for your contributions!

File: votes/0139-kennytm-as-tikv-committer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+* [tikv/crc64fast project](https://github.com/tikv/crc64fast)
+* [tikv/async-speed-limit project](https://github.com/tikv/async-speed-limit)
+
+I (@overvenus) hereby nominate @kennytm as TiKV Committer and call for a vote.
+
+## Result
+
+Approved by 2 binding votes, 1 non-binding votes.
+
+* overvenus (non-binding)
```
Comment (tisonkun) [inline votes/0139-kennytm-as-tikv-committer.md:19]:
```suggestion
* overvenus (non-binding)
* tisonkun (non-binding)
* skyzh (binding)
* BusyJay (binding)
```

**Review by sunxiaoguang — APPROVED** [review]
+1 binding

**Comment by sunxiaoguang** [comment]
LGTM

**Comment by tisonkun** [comment]
> +1 binding
> 
> Thanks for your contributions!

Thanks for your participant in this thread.

I think committees don't inherit the right as team maintainers. So as you're listed as tikv team committer, this vote is non-binding.

**Comment by lonng** [comment]
lgtm

**Review by lonng — APPROVED** [review]

**Comment by sunxiaoguang** [comment]
> > +1 binding
> > Thanks for your contributions!
> 
> Thanks for your participant in this thread.
> 
> I think committees don't inherit the right as team maintainers. So as you're listed as tikv team committer, this vote is non-binding.

Oops, Good to know more about governance in practice:)

File: votes/0139-kennytm-as-tikv-committer.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* tisonkun (non-binding)
+* skyzh (binding)
+* BusyJay (binding)
```
Comment (tisonkun) [inline votes/0139-kennytm-as-tikv-committer.md:22]:
```suggestion
* BusyJay (binding)
* sunxiaoguang (non-binding)
* lonng (non-binding)
```

**Comment by tisonkun** [comment]
@skyzh with 2 binding votes, we can accept this nomination. Would you mind to help on merging this PR?

**Comment by tisonkun** [comment]
GitHub Team membership updated. Congrats @kennytm!

**Pull request merged**